<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LibrariesWidget</class>
 <widget class="QWidget" name="LibrariesWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>161</width>
    <height>176</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QListWidget" name="listWidget">
       <property name="editTriggers">
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
       <property name="selectionBehavior">
        <enum>QAbstractItemView::SelectRows</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="addPushButton">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="mytypon.qrc">
           <normaloff>:/pictures/plus.png</normaloff>:/pictures/plus.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="removePushButton">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="mytypon.qrc">
           <normaloff>:/pictures/cross.png</normaloff>:/pictures/cross.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="upPushButton">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="mytypon.qrc">
           <normaloff>:/pictures/arrow_up.png</normaloff>:/pictures/arrow_up.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="downPushButton">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="mytypon.qrc">
           <normaloff>:/pictures/arrow_down.png</normaloff>:/pictures/arrow_down.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="mytypon.qrc"/>
 </resources>
 <connections/>
</ui>
